src/dotnet/GeneratorAgent/tests/Azure.Tools.GeneratorAgent.Tests/LocalLibraryGenerationServiceTests.cs

```diff
@@ -17,282 +10,209 @@ namespace Azure.Tools.GeneratorAgent.Tests
-                    appSettings ?? CreateAppSettings(),
-                    logger ?? CreateMockLogger().Object,
-                    processExecutionService ?? CreateMockProcessExecutionService().Object);
-            }
-
-            private string CreateUniqueTestDirectory(string baseName)
-            {
-                var uniqueId = Guid.NewGuid().ToString("N")[..8];
-                var directory = Path.Combine(Path.GetTempPath(), $"{baseName}-{uniqueId}");
-                Directory.CreateDirectory(directory);
-                _createdDirectories.Add(directory);
-                return directory;
-            }
-
-            private void CreateDefaultTypeSpecProject()
-            {
-                var packageJsonPath = Path.Combine(TypeSpecDir, "package.json");
-                File.WriteAllText(packageJsonPath, @"{
-                    ""name"": ""test-typespec"",
-                    ""version"": ""1.0.0"",
-                    ""devDependencies"": {
-                        ""@typespec/compiler"": ""latest"",
-                        ""@typespec/http-client-csharp"": ""latest""
-                    }
-                    }");
-                
-                var mainTspPath = Path.Combine(TypeSpecDir, "main.tsp");
-                File.WriteAllText(mainTspPath, @"import ""@typespec/http-client-csharp"";
-
-                    @service({
-                    title: ""Test Service"",
-                    })
-                    namespace TestService;
-                    ");
-
-                var tspConfigPath = Path.Combine(TypeSpecDir, "tspconfig.yaml");
-                File.WriteAllText(tspConfigPath, @"emit:
-  - ""@typespec/http-client-csharp""
-");
-            }
-
-            public void Dispose()
-            {
-                foreach (var directory in _createdDirectories)
-                {
-                    if (Directory.Exists(directory))
-                    {
-                        try
-                        {
-                            Directory.Delete(directory, true);
-                        }
-                        catch
-                        {
-                            // Intentionally ignored - cleanup failures shouldn't fail tests
-                        }
-                    }
-                }
-            }
-        }
-
-
         [Test]
         public void Constructor_WithValidParameters_ShouldInitializeCorrectly()
         {
-            using var fixture = new TestEnvironmentFixture();
-            
-            Assert.DoesNotThrow(() => fixture.CreateService());
+            // Arrange
+            var appSettings = CreateTestAppSettings();
+            var logger = NullLogger<LocalLibraryGenerationService>.Instance;
+            var processExecutionService = CreateMockProcessExecutionService().Object;
+
+            // Act & Assert
+            Assert.DoesNotThrow(() => new LocalLibraryGenerationService(appSettings, logger, processExecutionService));
```

Comment:
nit/nonblocking but I typically try to be selective about including tests that only test that something doesn't throw, sometimes it makes sense but other times it's just kind of like a smoke test. For constructors you'd probably see all the tests failing if the constructor starting throwing for just standard inputs